src/delimiter.js

```diff
@@ -354,38 +354,48 @@ const makeSqrtImage = function(height, options) {
     // Create a span containing an SVG image of a sqrt symbol.
     let span;
     let sizeMultiplier = options.sizeMultiplier;  // default
-    let spanHeight;
-    let viewBoxHeight;
+    let spanHeight = 0;
+    let texHeight = 0;
+    let viewBoxHeight = 0;
+
+    // We create viewBoxes with 80 units of "padding" above each surd.
+    // Then browser rounding error on the parent span height will not
+    // encroach on the ink of the viniculum. But that padding is not
+    // included in the TeX-like `height` used for calculation of
+    // vertical alignment. So texHeight = span.height < span.style.height.
 
     if (delim.type === "small") {
         // Get an SVG that is derived from glyph U+221A in font KaTeX-Main.
-        viewBoxHeight = 1000;  // from font
+        viewBoxHeight = 1080;  // 1000 unit glyph height + 80 unit padding.
```

Comment:
I think it be good to extract some of these values into constants, e.g.
```
const emBoxHeight = 1000;
const delimiterPadding = 80;

Reply:
Will do.